CMakeLists.txt

```diff
@@ -79,6 +80,12 @@ else()
     set(MLIO_STATIC_LIB TRUE)
 endif()
 
+if(MLIO_BUILD_IMAGE_READER)
```

Comment:
I don't think that defining an additional `USE_OPEN_CV` variable is necessary. You can already use `MLIO_BUILD_IMAGE_READER` for all conditionals

Reply:
addressed.